easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-foss-2016a.eb

```diff
@@ -0,0 +1,44 @@
+# -*- python -*-
+name = 'OpenFOAM'
+version = 'v1706'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a GPL-opensource C++ CFD-toolbox.
+This offering is not approved or endorsed by OpenCFD Ltd,
+producer and distributor of the OpenFOAM software via www.openfoam.com,
+and owner of the OPENFOAM trademark.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
```

Comment:
Honestly, I have no idea. I'm very new to easybuild but I understood that the configs need to specify a toolchain. The 2016b toolchang uses updated gcc etc.